packages/ui-react/src/components/IconButton/IconButton.module.scss

```diff
@@ -12,11 +13,19 @@
 
   cursor: pointer;
   opacity: 0.9;
-  transition: transform 0.1s ease;
+  transition: transform 0.1s ease, opacity 0.1s ease;
 
-  &:hover,
-  &:focus {
+  &:hover {
     transform: scale(1.1);
     opacity: 1;
   }
+
+  &:focus {
+    @include accessibility.accessibleOutline;
```

Comment:
This mixin is also added globally:
https://github.com/jwplayer/ott-web-app/blob/3329a0916bde27a66559ce492fd1af7013450311/packages/ui-react/src/styles/accessibility.scss#L41-L52

Reply:
You found another problem, indeed. The usage of the `:global` pseudo class name prevents this selector from working again. I have fixed this by not using the `@include` but explicitly defining the CSS rules...